cmd/xl-storage.go

```diff
@@ -403,23 +403,18 @@ func (s *xlStorage) CrawlAndGetDataUsage(ctx context.Context, cache dataUsageCac
 				// Bitrot check local data
 				if size > 0 && item.heal && opts.Bitrot {
 					s.waitForLowActiveIO()
-					err := s.VerifyFile(ctx, item.bucket, item.objectPath(), version)
-					switch err {
-					case errFileCorrupt:
-						res, err := objAPI.HealObject(ctx, item.bucket, item.objectPath(), oi.VersionID, madmin.HealOpts{
-							Remove:   healDeleteDangling,
-							ScanMode: madmin.HealDeepScan,
-						})
-						if err != nil {
-							if !errors.Is(err, NotImplemented{}) {
-								logger.LogIf(ctx, err)
-							}
-							size = 0
-						} else {
-							size = res.ObjectSize
+					// HealObject verifies bitrot requirement internally
```

Comment:
Do we still need `waitForLowActiveIO` whats the need for both `waitForLowActiveIO` and `waitForLowHTTPReq`

Reply:
that is meant for disk-level wait time @kerneltime - we need that to proceed with crawler healing checks. The tuning here for users who want to make disk healing faster.

Reply:
@kerneltime `s.waitForLowActiveIO`  is better because it is on a disk level and not on an API level.